backend/requirements/base.txt

```diff
@@ -0,0 +1,14 @@
+asgiref==3.4.1
+backports.zoneinfo==0.2.1
+Django==4.0.1
+django-filter==21.1
+djangorestframework==3.13.1
+importlib-metadata==4.10.0
+Markdown==3.3.6
+Pillow==9.0.0
+pkg_resources==0.0.0
+psycopg-binary==3.0.8
+psycopg2-binary==2.9.3
+pytz==2021.3
+sqlparse==0.4.2
+zipp==3.7.0
```

Comment:
I don't see `dotenv` here. Is it installed as a dependency (or included in the recent Django)?